app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<h4>Log in</h4>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do
+|f| %>
+<div class="mb-3">
+  <%= f.label :email, class:'form-label' %><br />
+  <%= f.email_field :email, autofocus: true, autocomplete: "email",
+  class:'form-control' %>
+</div>
+
+<div class="mb-3">
+  <%= f.label :password, class:'form-label' %><br />
+  <%= f.password_field :password, autocomplete: "current-password",
+  class:'form-control' %>
+</div>
```

Comment:
- [ ] Kindly fix your Homepage and add more styling for all pages
![Screenshot from 2022-03-25 16-21-10](https://user-images.githubusercontent.com/65860678/160141912-c09b8a80-b475-47f7-a89c-6f32ed07894f.png)
.

Reply:
This issue has been resolved

Reply:
- Well done for adding style. The design looks better.